examples/stork.toml

```diff
@@ -0,0 +1,5 @@
+provider = "Stork"
```

Comment:
This file should be committed in a different folder, with the code as it is, we cannot have price feed and stork configs in the same folder, because they are parsed twice, as `DynamicFeedConfig` and then as `StorkFeedConfig` too

Ideally it'll be better if both file types can coexists in the same folder, it'll require less DevOps overhead and it's much clear all the enabled feeds, the oracleType should be used to discriminate which type of config and the puller to use get the prices

Reply:
First we should separate them into different folders then process your idea in another pr to make this pr can be merged soon